### ISSUE TITLE
Puppet 6.17.0 fails to build

### ISSUE THREAD

**Issue by Sharpie** [Sharpie/puppet-build-experiment#5]
Build of `puppet-runtime` fails when the FFI gem tries to reference `pl-build-tools`:

```
 export GEM_HOME="/opt/puppetlabs/puppet/lib/ruby/gems/2.5.0" && \
export RUBYLIB="/opt/puppetlabs/puppet/lib/ruby/vendor_ruby:" && \
export PATH="/opt/pl-build-tools/bin:/opt/csw/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" && \
export PKG_CONFIG_PATH="/opt/puppetlabs/puppet/lib/pkgconfig:/usr/lib/arm-linux-gnueabihf/pkgconfig" && \
: nothing to extract
touch rubygem-ffi-unpack
touch rubygem-ffi-patch
[ -d ./ ] || mkdir -p ./
export GEM_HOME="/opt/puppetlabs/puppet/lib/ruby/gems/2.5.0" && \
export RUBYLIB="/opt/puppetlabs/puppet/lib/ruby/vendor_ruby:" && \
export PATH="/opt/pl-build-tools/bin:/opt/csw/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" && \
export PKG_CONFIG_PATH="/opt/puppetlabs/puppet/lib/pkgconfig:/usr/lib/arm-linux-gnueabihf/pkgconfig" && \
cd ./ && \
sed -i '0,/ensure_required_ruby_version_met/b; /ensure_required_ruby_version_met/d' /opt/pl-build-tools/lib/ruby/2.1.0/rubygems/installer.rb
sed: can't read /opt/pl-build-tools/lib/ruby/2.1.0/rubygems/installer.rb: No such file or directory
make[1]: *** [Makefile:966: rubygem-ffi-configure] Error 2
make[1]: Leaving directory '/var/tmp/tmp.Pz0Z0QcQZo'
Local command ((cd /var/tmp/tmp.Pz0Z0QcQZo; /usr/bin/make )) failed.
```

**Comment by Sharpie** [comment]
This looks like it was caused by further patches to force FFI to build with the ancient Ruby version used by `pl-build-tools`. The Rasbian builds skip using `pl-build-tools` and just use the host Ruby:

https://github.com/puppetlabs/puppet-runtime/commit/2c8f42ab74801b48e075f48e244150d73e315274

**Comment by Sharpie** [comment]
Fixed by adea8b2
